wdl-analysis/src/stdlib.rs

```diff
@@ -1651,26 +1758,150 @@ pub static STDLIB: LazyLock<StandardLibrary> = LazyLock::new(|| {
                         .parameter(PrimitiveType::Integer)
                         .parameter(PrimitiveType::Integer)
                         .ret(PrimitiveType::Integer)
+                        .definition(
```

Comment:
For polymorphic signatures, let's `const`-ize the definition just above the declaration and refer to that instead of duplicating.

Reply:
Some of the polymorphic signatures have specific markup for the arguments. Those should stay as they are (e.g., a stdlib function that can take an object, map, and dict). For ones like this one though, where the content stays the same across the polymorphic signatures, let's `const`.

Reply:
What if we use those parameter and return types from function signatures  instead of a generic `Float | Int` 🤔